frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/UnmanagedBranchNotificationBuilder.java

```diff
@@ -0,0 +1,96 @@
+package com.virtuslab.gitmachete.frontend.ui.impl.table;
+
+import static com.virtuslab.gitmachete.frontend.datakeys.DataKeys.typeSafeCase;
+import static com.virtuslab.gitmachete.frontend.defs.ActionIds.SLIDE_IN_UNMANAGED_BELOW;
+import static com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString;
+import static io.vavr.API.$;
+import static io.vavr.API.Case;
+import static io.vavr.API.Match;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationAction;
+import com.intellij.notification.NotificationType;
+import com.intellij.openapi.actionSystem.ActionManager;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.CommonDataKeys;
+import com.intellij.openapi.actionSystem.DataContext;
+import com.intellij.openapi.actionSystem.Presentation;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vcs.VcsNotifier;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import com.virtuslab.gitmachete.backend.api.IGitMacheteRepositorySnapshot;
+import com.virtuslab.gitmachete.frontend.datakeys.DataKeys;
+import com.virtuslab.gitmachete.frontend.defs.ActionPlaces;
+
+@RequiredArgsConstructor
+public class UnmanagedBranchNotificationBuilder {
```

Comment:
I'd call it a `...Factory`, as `Builder` implies sth closer to Lombok `@Builder`

Reply:
yeah ofc, I am aware - i am still working on this part

Reply:
Ok 🤐 then